src/main/java/dk/aau/p4/abaaja/Lib/Visitors/SymbolTableVisitor.java

```diff
@@ -472,18 +464,62 @@ else if ((funcSymbol.get_types().size() == 0 && node.get_paramExps().size() != 0
     }
 
     // Is implemented in visitFuncDecNode to check the type correctness of the expression
```

Comment:
Is this comment still accurate?

Reply:
Hmm, that should have been changed to the following:
```suggestion
    // Fully Implemented
```